src/EventStore.Core/EventStoreHostedService.cs

```diff
@@ -30,7 +30,9 @@ public abstract class EventStoreHostedService<TOptions> : IHostedService where T
 					Path.Combine(Locations.DefaultConfigurationDirectory, DefaultFiles.DefaultConfigFile),
 					MutateEffectiveOptions);
 				if (Options.Help) {
-					Console.WriteLine("Options:");
+					Console.WriteLine("EventStore version {0} ({1}/{2}, {3})",
```

Comment:
```suggestion
					Console.WriteLine("EventStoreDB version {0} ({1}/{2}, {3})",
```

Reply:
I've updated it for --help and --version